codegen/src/test/resources/software/amazon/awssdk/codegen/poet/plugins/TestPlugin1.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.codegen.poet.plugins;
+
+import software.amazon.awssdk.core.SdkPlugin;
+import software.amazon.awssdk.core.SdkServiceClientConfiguration;
+
+public class TestPlugin1 implements SdkPlugin {
```

Comment:
Can we call it `InternalPlugin1` or `InternalTestPlugin1` to make it even more obvious what the role of the plugin is?

Also, nit: our own classes are marked with the api annotations and this would be as well, `@SdkInternalApi`

Reply:
Okay, will modify this in the next revision.